tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -800,25 +807,25 @@ Duration <DURATION>"
 `;
 
 exports[`inflight_ref_explicit_ops.w 1`] = `
-"error: Unable to qualify which operations are performed on 'this.myQueue' of type 'Queue'. This is not supported yet.
+"error: Cannot qualify access to a lifted object of type \\"Queue\\" (see https://github.com/winglang/wing/issues/76 for more details)
    --> ../../../examples/tests/invalid/inflight_ref_explicit_ops.w:13:12
    |
 13 |     return this.myQueue;
-   |            ^^^^^^^^^^^^ Unable to qualify which operations are performed on 'this.myQueue' of type 'Queue'. This is not supported yet.
+   |            ^^^^^^^^^^^^ Cannot qualify access to a lifted object of type \\"Queue\\" (see https://github.com/winglang/wing/issues/76 for more details)
 
 
-error: Unable to qualify which operations are performed on 'this.b' of type 'Bucket'. This is not supported yet.
+error: Cannot qualify access to a lifted object of type \\"Bucket\\" (see https://github.com/winglang/wing/issues/76 for more details)
    --> ../../../examples/tests/invalid/inflight_ref_explicit_ops.w:34:13
    |
 34 |     let x = this.b;
-   |             ^^^^^^ Unable to qualify which operations are performed on 'this.b' of type 'Bucket'. This is not supported yet.
+   |             ^^^^^^ Cannot qualify access to a lifted object of type \\"Bucket\\" (see https://github.com/winglang/wing/issues/76 for more details)
 
 
-error: Capturing collection of preflight classes is not supported yet (type is 'Array<Bucket>')
-   --> ../../../examples/tests/invalid/inflight_ref_explicit_ops.w:47:18
+error: Cannot qualify access to a lifted object of type \\"Bucket\\" (see https://github.com/winglang/wing/issues/76 for more details)
+   --> ../../../examples/tests/invalid/inflight_ref_explicit_ops.w:47:13
```

Comment:
Aren't we checking capturing of preflight classes in collections and handling this anymore? Why is it complaining about the qualifying thing here and not saying something about this being a collection of preflight objects (different issue).

Reply:
Removed the special case to handle collections because you can now lift a specific element within a collection.